enhancements/olm/olm-managed-operator-metrics.md

```diff
@@ -110,26 +103,58 @@ A `ServiceMonitor` object must be created in the namespace mentioned above and p
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: prometheus-k8s
+  namespace: placeholder
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - services
+  - endpoints
+  - pods
+  verbs:
+  - get
+  - list
+  - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: prometheus-k8s
+  namespace: placeholder
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: prometheus-k8s
+subjects:
+- kind: ServiceAccount
+  name: 'prometheus-k8s',
+  namespace: 'openshift-monitoring',
+```
 
 #### Fulfilling Namespace and RBAC Requirements
 
-OLM itself will not be responsible for fulfilling the Namespace and RBAC requirements. Instead, OLM will define how operator authors can provide OpenShift Console with the information required to fulfill these requirements.
+The OpenShift Monitoring Requirements mentioned above can be met in one of two ways:
 
-OLM [building-your-csv](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md) documentation will be updated to describe two new supported annotations:
+- By using annotations that instructs OpenShift Console to generate the required Namespace and RBAC Requirements
+- By packaging the RBAC [mentioned above](#####rbac-requirements) within the Operator Bundle and having a cluster admin install the operator in a namespace that is both prefixed with `openshift-` and has the `openshift.io/cluster-monitoring=true` label.
 
-- The `operatorframework.io/suggested-namespace` annotation. When the `operatorframework.io/suggested-namespace` annotation is present the UI will highlight the suggested namespace when installing the operator. OLM itself will not require that the operator is deployed in the namespace defined by the `operatorframework.io/suggested-namespace` annotation.
+##### Fulfilling the Namespace and RBAC Requirements via Console
 
-- The `operatorframework.io/cluster-monitoring=true` annotation. When
-  this annotation is set to `true`, the OpenShift Console will update
-  the namespace that the operator is being deployed to with the
-  `openshift.io/cluster-monitoring=true` label. When this annotation
-  is present, the UI will update the OpenShift Monitoring Prometheus
-  Operator ServiceAccount with the appropriate RBAC privileges for the
-  given namespace as well, allowing operators to be scraped by the
-  OpenShift Monitoring Prometheus Operator.
+OLM  [building-your-csv](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md) documentation will be updated to describe two new supported annotations:
 
->Note: There is no work required by the OLM team to implement this support.
+- The `operatorframework.io/suggested-namespace` annotation.
```

Comment:
I'm surprised that the namespace is `operatorframework.io` (upstream), when its specific to OpenShift Console.

Reply:
We might want to pick this up later though in OLM. The use case is still valid.

Reply:
fair enough @dmesser ! 

If console is upstream, should the namespace be qualified by it? e.g. `console.operatorframework.io`?

Reply:
@njhale one way I could imagine this to work is that when OLM decides in which namespace the Operator lands as part of a global singleton deployment, it takes this input into account and uses it instead of randomly creating a new namespace. Kind of depends on how we intend that process to look like with the new APIs.